dcp_prototype/frontend/src/components/exploreData.js

```diff
@@ -4,21 +4,28 @@ import StyledButton from "./styledButton"
 import LoginSignup from "./login-signup"
 
 import { api_url, cxg_url } from "../globals"
+import {useAuth0} from "../contexts/auth0Context";
 
 const ExploreData = ({ project, files, isAuthenticated }) => {
   const matrixAvailable = (files && files.length && isAuthenticated) === true
   const cxgAvailable = project.cxg_enabled
+    const { getTokenSilently } = useAuth0()
 
   return (
   <Box sx={{mb: 2}}>
     <Flex sx={{mb: 1}}>
       <StyledButton label="Download matrix"
                     disabled={!matrixAvailable}
-                    onclick={() => fetch(`${api_url}/files/${files[0].id}`)
-                                    .then(response => response.json())
-                                    .then(resultData => {
-                                      window.open(resultData.url)
-                                    })}>
+                    onclick={() => getTokenSilently().then(accessToken =>
```

Comment:
Would be nice to define this as a named function (same goes for other anons). I'll do this refactoring as I write the E2E tests for the frontend